File: examples-sdk/swift/304-rag-citation-extraction/main.swift
// 304-rag-citation-extraction — Swift SDK equivalent
// Compile: vil compile --from swift --input 304-rag-citation-extraction/main.swift --release

let server = VilServer(name: "rag-citation-extraction", port: 3113)
let rag_citation = ServiceProcess(name: "rag-citation")
rag_citation.endpoint(method: "POST", path: "/cited-rag", handler: "cited_rag_handler")
server.service(rag_citation)
server.compile()

File: examples-sdk/csharp/304-rag-citation-extraction/Main.cs
// 304-rag-citation-extraction — C# SDK equivalent
// Compile: vil compile --from csharp --input 304-rag-citation-extraction/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("rag-citation-extraction", 3113);
var rag_citation = new ServiceProcess("rag-citation");
rag_citation.Endpoint("POST", "/cited-rag", "cited_rag_handler");
server.Service(rag_citation);
server.Compile();
